MixItUp.WPF/Controls/Chat/ChatMessageHeaderControl.xaml.cs

```diff
@@ -14,10 +17,24 @@ namespace MixItUp.WPF.Controls.Chat
     /// </summary>
     public partial class ChatMessageHeaderControl : UserControl
     {
+        private static int whisperCount = 0;
+        private Dictionary<string, int> whisperMap = new Dictionary<string, int>();
```

Comment:
Would this need to be a static object so that it keeps track of all users for every whisper?